content/1.5/docs/installation/orchestrator-crd-versions/v1alpha3.md

```diff
@@ -3,4 +3,72 @@ title: "CRD Version v1alpha3"
 date: 2025-04-20
 ---
 
-TBD
\ No newline at end of file
+The Go-Based Operator was introduced in Orchestrator 1.5 since the helm-based operator is currently in maintenance mode.
+Also, with major changes to the CRD, the v1alpha3 version
+of [Orchestrator CRD](https://github.com/rhdhorchestrator/orchestrator-go-operator/blob/release-1.5/config/crd/bases/rhdh.redhat.com_orchestrators.yaml)
+was introduced and is not backward compatible.
+
+In this version, the CRD field structure has completely changed with most fields either removed or renamed and
+restructured.
+To see more information about the CRD fields, check out the
+full [Parameter list](https://github.com/rhdhorchestrator/orchestrator-go-operator/blob/release-1.5/docs/crd/README.md).
```

Comment:
Some of the parameters don't have description. It would be nice to add it where it is missed.
(unrelated to this PR of course)